spec.html

```diff
@@ -23352,6 +23351,12 @@ <h1>NaN</h1>
       <h1>undefined</h1>
       <p>The value of `undefined` is *undefined* (see <emu-xref href="#sec-ecmascript-language-types-undefined-type"></emu-xref>). This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
     </emu-clause>
+
+    <emu-clause id="sec-global-this">
+      <h1>globalThis</h1>
```

Comment:
Properties are usually listed in (case-insensitive) alphabetical order, which would put this up before "Infinity".